app/models/like.rb

```diff
@@ -0,0 +1,12 @@
+class Like < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ]  @Yosefgeda great work adding associations in the like model, however, you reference the user wrongly, instead, you should use the class and author eg `belongs_to :author, class_name: 'User'`